.github/workflows/build.yml

```diff
@@ -91,7 +87,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Build docker image
         env:
-          SAMPLE_REGISTRY: ghcr.io/dapr/samples
+          SAMPLE_REGISTRY: ghcr.io/${{ github.repository_owner }}/samples
```

Comment:
You need to lowercase this or it won't work for people who have mixed case usernames on GitHub like, say, ItalyPaleAle :) 

Example: https://github.com/dapr/dapr/blob/3713a058c7c64455ffa8d96d1bc173b7e64f96d2/.github/workflows/dapr.yml#L84-L90